docs/source/release_notes.rst

```diff
@@ -6,6 +6,7 @@ Release Notes
 Future Release
 ==============
     * Enhancements
+      * Add ``series_library`` attribute (:pr:`2257`) 
     * Fixes
     * Changes
```

Comment:
should we mention schema change here?

Reply:
yah we can mention it with this header
```
Breaking Changes
++++++++++++++++
```

Reply:
we haven't mentioned serialization schema changes in the past

Reply:
True, but we could start doing it now.

Reply:
@sbadithe @gsheni do we want to add this?  Rest of PR seems ready to go

Reply:
If it's an easy lift, I would do it.

Reply:
Sure, should I push a commit?

Reply:
Sure